### PR TITLE
Browser Monitoring Edit

### PR DESCRIPTION
**Pull request by alai97**
## Motivation

#rum-docs Slack

## Changes

Adds copyedits to the Browser Monitoring doc index page, moves some content into a guide.

## Preview

https://docs-staging.datadoghq.com/alai97/rum-browser-and-browser-premium-sampling-guide/real_user_monitoring/browser/ (the link to the guide page does not work here yet).

https://docs-staging.datadoghq.com/alai97/rum-browser-and-browser-premium-sampling-guide/real_user_monitoring/guide/sampling-browser-and-browser-premium

## Testing

- [ ] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/rum/README.md [hunk 1]
```diff
@@ -448,6 +450,8 @@ datadogRum.init({
 
 </details>
 
+{{< img src="real_user_monitoring/browser/example-initialization-snippet.mp4" alt="Example initialization snippet for a browser application with custom percentages" video="true" width="80%" >}}
```
Comment (alai97) [inline packages/rum/README.md:453]:
```suggestion
{{< img src="real_user_monitoring/browser/example-initialization-snippet.mp4" alt="Example initialization snippet for a browser application with custom percentages" video="true" width="100%" >}}
```

File: packages/rum/README.md [hunk 2]
```diff
@@ -392,6 +392,8 @@ Two initialization parameters are available to control how the session is tracke
 - `sampleRate` controls the percentage of overall sessions being tracked. It defaults to `100%`, so every session is tracked by default.
 - `premiumSampleRate` is applied **after** the overall sample rate, and controls the percentage of sessions tracked as Browser Premium. It defaults to `100%`, so every session is tracked as Browser Premium by default.
 
+{{< img src="real_user_monitoring/browser/browser-initialization-snippet.png" alt="Default initialization snippet for a browser application" style="width:80%;" >}}
```
Comment (alai97) [inline packages/rum/README.md:395]:
```suggestion
{{< img src="real_user_monitoring/browser/browser-initialization-snippet.png" alt="Default initialization snippet for a browser application" style="width:100%;" >}}
```

**Review by maycmlee — APPROVED** [review]

File: packages/rum/README.md [hunk 3]
```diff
@@ -485,16 +487,18 @@ A service is an independent, deployable code repository that maps to a set of pa
...
-After the Datadog browser RUM SDK is initialized, you can access the internal context of the SDK. This allows you to access:
+After the Datadog Browser SDK is initialized, you can access the internal context of the SDK. 
 
-| Attribute      | Description                                                   |
-| -------------- | ------------------------------------------------------------- |
-| application_id | ID of the application                                         |
-| session_id     | ID of the session                                             |
-| user_action    | Object containing action ID (or undefined if no action found) |
-| view           | Object containing details about the current view event        |
+You can access the following attributes:
```
Comment (alai97) [inline packages/rum/README.md:492]:
```suggestion
You can explore the following attributes:
```

**Review by liywjl — APPROVED** [review]

File: packages/rum/README.md [hunk 4]
```diff
@@ -447,9 +447,11 @@ datadogRum.init({
 ```
 
 </details>
-
+   
 The `premiumSampleRate` is a percentage of `sampleRate`. If you set `sampleRate` to 60 and `premiumSampleRate` to 50, 40% of sessions are dropped, 30% of sessions are collected as Browser and 30% of sessions are collected as Browser Premium.
 
+{{< img src="real_user_monitoring/browser/example-initialization-snippet.mp4" alt="Example initialization snippet for a browser application with custom percentages" video="true" width="100%" >}}
```
Comment (BenoitZugmeyer) [inline packages/rum/README.md:453]: **🔨 warning:** ‏This won't render well in Github. AFAIK, you can't add a video to a GitHub README... maybe add just a link to it?

**Review by BenoitZugmeyer — CHANGES_REQUESTED** [review]

File: packages/rum/README.md (hunk 4 above)
Comment (BenoitZugmeyer) [reply packages/rum/README.md:453]: It looks like a lot of documentation edits are about clarifying the sample rate options. Maybe having a dedicated page related to sample rate configuration could be an interesting option, so we don't have github-related limitations.

Comment (alai97) [reply packages/rum/README.md:453]:
```suggestion
```

File: packages/rum/README.md [hunk 5]
```diff
@@ -2,36 +2,36 @@
 
 ## Overview
 
-Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users. To collect events, add the RUM Browser SDK to your browser application and configure what data is collected using initialization parameters.
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application's individual users. To collect events, add the Datadog Browser SDK to your browser application and configure what data is collected using initialization parameters.
```
Comment (BenoitZugmeyer) [inline packages/rum/README.md:5]: **❓ question:** ‏Why changing the name? We have two "Browser SDK", Logs and RUM. This doc is about RUM, so IMO it should be `RUM Browser SDK` (or eventually `Datadog RUM Browser SDK`)

File: packages/rum/README.md [hunk 6]
```diff
@@ -376,10 +376,10 @@ init(configuration: {
 
 ### Browser and Browser Premium sampling configuration
```
Comment (BenoitZugmeyer) [inline packages/rum/README.md:377]: **💬 suggestion:** ‏Move this whole section into the new sampling guide. This way, everything is documented at the same place. We can just link to it directly.

Comment (jamiemilstein) [reply packages/rum/README.md:377]: i like that idea

File: packages/rum/README.md [hunk 7]
```diff
@@ -334,7 +334,7 @@ Telemetry data (such as errors and debug logs) about SDK execution is sent to Da
 **Type:** List<br/>
 A list of request origins ignored when computing the page activity. See [How page activity is calculated][16].
 
-Options that must have matching configuration when also using the Logs Browser SDK:
+Options that must have matching configuration when you using the Datadog Browser SDK for logs:
```
Comment (jamiemilstein) [inline packages/rum/README.md:337]: "you using" feels like a typo/gramatically incorrect. you're?

**Comment by jamiemilstein** [comment]
On the guides page, I don't think [this title](https://a.cl.ly/bLu51WlA) makes sense for it. It's more about the setup

File: packages/rum/README.md (hunk 7 above)
Comment (alai97) [reply packages/rum/README.md:337]:
```suggestion
Options that must have matching configuration when you are using the Datadog Browser SDK for logs:
```

**Comment by alai97** [comment]
@BenoitZugmeyer feel free to merge!

File: packages/rum/README.md [hunk 8]
```diff
@@ -572,3 +472,4 @@ window.DD_RUM && window.DD_RUM.getInternalContext() // { session_id: "xxxx", app
 [18]: https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options
 [19]: https://docs.datadoghq.com/getting_started/tagging/using_tags
 [20]: https://docs.datadoghq.com/real_user_monitoring/frustration_signals/
+[21]: https://docs.datadoghq.com/real_user_monitoring/guide/sampling-browser-and-browser-premium/
```
Comment (bcaudan) [inline packages/rum/README.md:475]: **❗ warning:** ‏this doc must be merged before merging this PR

**Comment by BenoitZugmeyer** [comment]
I moved the Browser SDK renaming to a separate PR #1707 as I want to submit it to PM approval beforehand.

File: packages/rum/README.md (hunk 8 above)
Comment (alai97) [reply packages/rum/README.md:475]: My linked PR (https://github.com/DataDog/documentation/pull/15038) has been merged!

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**